components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -513,6 +490,44 @@ bool PublisherInfoDatabase::InsertOrUpdateActivityInfos(
   return transaction.Commit();
 }
 
+bool PublisherInfoDatabase::GetExcludedList(
+    ledger::PublisherInfoList* list) {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  CHECK(list);
+
+  bool initialized = Init();
+  DCHECK(initialized);
+
+  if (!initialized) {
+    return false;
+  }
+
+  // We will use every attribute from publisher_info
+  std::string query = "SELECT * FROM publisher_info "
```

Comment:
why do we need activity info? We just need to list all publishers in publisher_info that has exclude set to 1

Reply:
yep, we can simplify and just use `publisher_info`, done